service/src/main/java/uk/gov/hmcts/reform/fpl/enums/OrderStatus.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.fpl.enums;
+
+public enum OrderStatus {
```

Comment:
Could use lombok here with a `@RequiredArgsContructor` and `@Getter` to reduce boiler plate